docs/contributing/community-slack.md

```diff
@@ -35,10 +35,10 @@ For other cases, other channels might be preferable:
 
 1. If you suspect a security issue in Gradle Build Tool, plugins, its infrastructure, or other components,
    please avoid discussing it in public.
-   Follow the [Gradle Vulnerability Disclosure Policy](../dotgithub/SECURITY/README.md) so that we can triage the issue and protect other Gradle users.
-3. If you want to report a confirmed and reproducible issue with Gradle Build Tool or other component,
+   Follow the [Gradle Vulnerability Disclosure Policy](dotgithub/SECURITY/README.md) so that we can triage the issue and protect other Gradle users.
```

Comment:
Oops

Reply:
It's still not fixed, I will patch it in my next PR. I think the correct path would be `../../dotgithub/SECURITY/README.md`